style.css

```diff
@@ -0,0 +1,619 @@
+  padding: 30px 20px;
+}
+
+/* ||  ====== Patterns ======= || */
+
+.pattern-2 {
+  position: absolute;
+  background-image: url('images/capsPat.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 60px;
+  height: 60px;
+  background-size: cover;
+}
+
+#desc-pat-tr {
+  top: -62px;
+  right: 20px;
+  background-position: center 20px;
+}
+
+hr {
+  width: 25px;
+  background-color: #ec5242;
+  border: none;
+  height: 2px;
+  margin-top: 7px;
+}
+
+/* || ===== Main Program ======  */
+
+#programs {
+  background-image: url(images/pat-m.png);
+  background-repeat: repeat;
+  background-color: #272a31fc;
+  color: #d3d3d3;
+  justify-content: space-around;
+  padding: 40px 0;
+}
+
+.progs {
+  row-gap: 10px;
+  margin: 30px 0;
+}
+
+.pro-item {
+  background-color: rgba(255, 255, 255, 0.05);
+  padding: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+#abt-partners-ph,
+#partners-ph {
+  background-color: #272a31;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.partners-list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  margin-top: 18px;
+  row-gap: 10px;
+}
+
+.partners {
+  min-width: 60px;
+  width: 26%;
+  justify-content: center;
+}
+
+#footer { background-color: #fff; }
+
+#partners { text-align: center; }
+
+#host-ph {
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 0;
+  gap: 20px;
+}
+
+/* || ======= Utilities ======== || */
+
+.fw { width: 100%; }
+
+.pro-item:hover {
+  border: 1px solid #d3d3d3;
+}
+
+.menu {
+  width: 42px;
+  height: 42px;
+  background-color: transparent;
+  border: none;
+  background-image: url(images/menu.svg);
+}
+
+.navlink {
+  color: #272a31;
+  padding: 6px 8px;
+}
+
+#nav-link-tr-1,
+.navlink:active { color: #ec5242; }
+
+#nav-link-tr-6 { border: 4px solid #ec5242; }
+
+.btn {
+  height: 64px;
+  width: 72%;
+  background-color: #ec5242;
+  border: none;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+.mob-toggle {
+  width: 100%;
+  height: 52px;
+  border-radius: 5px;
+  border: 1px solid #d3d3d3;
+  background-color: #fff;
+  background-image: url('images/more.png');
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+.btn-less { background-image: url('images/less.png'); }
+
+/* || Overlay Menu */
+
+.overlay-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  background-color: #272a31;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+  font-size: 2em;
+  line-height: 1.5em;
+  font-weight: 600;
+  overflow: hidden;
+  padding: 30px 0;
+  transition: all 2s;
+}
+
+#close {
+  height: 24px;
+  width: 24px;
+  align-self: flex-end;
+  margin-right: 32px;
+  margin-bottom: 32px;
+}
```

Comment:
- [ ]  You should use background that is transparent or the same color/pattern as the top section background. what I suggest here is to make it more transparent than it is.
![image](https://user-images.githubusercontent.com/43877689/227643849-0d1c38fc-b15d-40b0-9965-62ce5a0acce0.png)